src/include/partitioning/custom_part_dialogs.rb

```diff
@@ -1582,7 +1582,7 @@ def SubvolHandling(old, new)
         )
       )
 
-      if Mode.installation()
+      if Mode.installation() && snapshots_supported?(new)
```

Comment:
why it use new and not self?...ah, great variable name. Is possible to rename variable to something like old_partitioning and new_partitioning?